### ISSUE TITLE
Change H1 place/behaviour of motion-list

### ISSUE THREAD

**Issue by Elblinator** [OpenSlides/openslides-client#6136]
**Describe the bug**
In a motion list the H1 structure element is not on the navigatable link from the motion but on the title of the motion.

So the title from the motion can be navigated to (and hence, be read out loud)
But if you navigated to the title (with the structure elements) you have to go one back to be able to navigate to the link for the motion

**How to Reproduce**
1. [use the NVDA screen reader and navigate via the headers]
2. have at least one motion
3. go to the lotion list
4. navigate to the header 1 of the motion
5. press enter
6. nothing happens
7. press shift + tab
8. press enter 
9. you navigate to the motion

**Wanted behavior**
If you navigate to the motion title (which could be read out loud) then you should be able to just press enter to navigate into the motion

**Additional context**
The H1 element probably has be removed and/or moved to the navigateable 'a' element

The visual appearance should not change!!
